src/main/kotlin/com/sourcegraph/cody/config/migration/SettingsMigration.kt

```diff
@@ -64,6 +64,9 @@ class SettingsMigration : Activity {
 
     DeprecatedChatLlmMigration.migrate(project)
     ChatTagsLlmMigration.migrate(project)
+    RunOnceUtil.runOnceForApp("CodyMigrateChatHistory") {
```

Comment:
I think this should be `runOnceForProject`.
I think few other using `runOnceForApp` are incorrect too, could you maybe fix them by the way?
The only one which should use `runOnceForApp` is - I believe - `CodyApplicationSettingsMigration` as you can see it's not using `project` at all.

Reply:
Ah nice. Updated.